frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -174,16 +174,12 @@ const OrganizationView: React.FunctionComponent<OrganizationViewProps> = ({ orga
           <Col>
             {organization.persons &&
               organization.persons.map((person: IContactPerson, index: number) => (
-                <>
-                  <Link
-                    to={'/contact/P' + person.id}
-                    data-testid="contact-organization-person"
-                    key={'org-person-' + index}
-                  >
+                <span>
```

Comment:
@FuriousLlama I think you need to re-add the key here otherwise react will throw a warning.